modules/vector-search/pages/vector-search.adoc

```diff
@@ -0,0 +1,50 @@
+= Use Vector Search for AI Applications
+:page-topic-type: concept
+:description: Use Couchbase Capella's Vector Search features to add fast and accurate semantic search to your applications.
+
+[abstract]
+{description}
+
+== About Vector Search
+
+Vector Search builds on Couchbase Capella's xref:search:search.adoc[Search Service] to provide vector index support.
+You can use these new Vector Search indexes for Retrieval Augmented Generation (RAG) with an existing Large Language Model (LLM). 
+
+Using Vector Search, Capella, an embedding model, and your chosen LLM, you can develop AI applications while giving context and up-to-date information from your own data.
```

Comment:
Agree we don't want to get into the weeds...but it feels like we could use a bit more context around RAG, LLM, and embedding models. I would not hold up GA for this, but I think some simple definitions of these things and the role they play in AI would be helpful, particularly to new developers